charts/rancher-k3s-upgrader/0.6.0/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+name: rancher-k3s-upgrader
+description: Enables a k3s or rke2 cluster to update itself by reacting to Plan CRs.
+  Users do not need to manually upgrade this app. It will be automatically upgraded to the latest version when upgrading a cluster.
+home: https://github.com/rancher/system-charts/blob/dev-v2.7.11/charts/rancher-k3s-upgrader
+sources:
+  - "https://github.com/rancher/system-charts/blob/dev-v2.7.11/charts/rancher-k3s-upgrader"
```

Comment:
What will be the ultimate branch for hosting this chart, will it be `release-v2.7`?

Reply:
So the chart version already exists in `dev-v2.8` and above but with different rancher requirements - will the 2.7.11 branch get merged back into 2.7? If so I'm happy to update it to just `release-v2.7`.

Reply:
I think about it again, I think the branch `dev-v2.7.11` will be there forever as a reference to the source, so it should be ok to keep the file as it is.